changelog.md

```diff
@@ -4,6 +4,7 @@
 - [FIXED] Issue with query generation in MSSQL, an identifier was not escaped [#6686] (https://github.com/sequelize/sequelize/pull/6686)
 - [FIXED] GroupedLimit when foreignKey has a field alias
 - [FIXED] groupedLimit.through.where support
+- [FIXED] Issue with overrriding custom methods with association mixins [#6682] (https://github.com/sequelize/sequelize/issues/6682)
```

Comment:
Should also mention that there are now new methods on `Association` objects, maybe even document those (do we have documentation for `Association` classes?)